src/aasthaJJun20/NumbersAndArrays.java

```diff
@@ -0,0 +1,79 @@
+package aasthaJJun20;
+
+public class NumbersAndArrays {
+	//This method is written just to display the array elements in output
+	void displayArray(int[] input) {
+		System.out.print("Input Array : { ");
+		for(int index = 0; index < input.length; index++) {
+			System.out.print(input[index]+" ");;
+		}
+		System.out.println("}");
+	}
+	
+	// Program 1: Find the missing number in integer array of 1 to 10?
+	void findMissingNumberFromArray(int[] input) {
+		displayArray(input);
+		int actualSum = 0, differedSum = 0;
+		for (int index = 1; index <= 10; index++) {
+			actualSum += index;
```

Comment:
@AasthaJJun20  @NikhilMJun20  ,To sum of first N number, we can use formula to impress interviewer.

n*(n+1)/2 --> (10*11)/2 = 55 :)

Reply:
Yes Krishna, this formula will make the logic simple and short as well. Thanks